use-timescale/compression/about-compression.md

```diff
@@ -11,189 +11,158 @@ import CompressionIntro from 'versionContent/_partials/_compression-intro.mdx';
+data. Timescale creates and uses custom indexes to incorporate the `segmentby` 
+and `orderby` parameters during compression which are used when reading compressed data.
+More on this in the next section.
+</Highlight>
 
-```sql
-ALTER TABLE  example
-  SET (timescaledb.compress,
-       timescaledb.compress_orderby = 'device_id, time');
-```
+Based on the previous schema, filtering of data should happen over a certain time period and analytics are done on device granularity. This pattern of data access lends itself to organizing the data layout suitable for compression.
 
-Using those settings, the compressed table now shows each measurement in
-consecutive order, and the `cpu` values show a trend. This table compresses much
-better:
+### Segmenting and ordering.
 
-|time|device_id|cpu|disk_io|energy_consumption|
-|-|-|-|-|-|
-|[12:00:02, 12:00:01, 12:00:02, 12:00:01 ]|[1, 1, 2, 2]|[88.2, 88.6, 300.5, 299.1]|[20, 25, 30, 40]|[0.8, 0.85, 0.9, 0.95]|
-
-Putting items in `orderby` and `segmentby` columns often achieves similar
-results. In this same example, if you set it to segment by the `device_id`
-column, it has good compression, even without setting `orderby`. This is because
-ordering only matters within a segment, and segmenting by device means that each
-segment represents a series if it is ordered by time. So, if segmenting by an
-identifier causes segments to become too small, try moving the `segmentby`
-column into a prefix of the `orderby` list.
-
-You can also use ordering to increase query performance. If a query uses similar
-ordering as the compression, you can decompress incrementally and still return
-results in the same order. You can also avoid a `SORT`. Additionally, the system
-automatically creates additional columns to store the minimum and maximum value
-of any `orderby` column. This way, the query executor looks at this additional
-column that specifies the range of values in the compressed column, without
-first performing any decompression, to determine whether the row could possibly
-match a time predicate specified by the query.
-
-## Insert historical data into compressed chunks
-
-In TimescaleDB 2.3 and later, you can insert data directly into compressed
-chunks. When you do this,the data that is being inserted is not compressed
-immediately. It is stored alongside the chunk it has been inserted into, and
-then a separate job merges it with the chunk and compresses it later on.
-
-[decompress-chunks]: /use-timescale/:currentVersion:/compression/decompress-chunks
-[manual-compression]: /use-timescale/:currentVersion:/compression/manual-compression/
+Segmenting the compressed data should be based on the way you access the data. Basically, you want to segment your data in such a way that you can make it easier for your queries to fetch the right data at the right time. That is to say, your queries should dictate how you segment the data so they can be optimized and yield even better query performance.
```

Comment:
This is a little confusing because it starts using the term "segmenting" before explaining what this is. Probably it should go like this:
1) why you have to order data for compression
2) how you do this in Timescale (orderby column)
3) but we also have a segmentby and it's useful in the following ways

By the way, why do we use segmentby at all? The only thing that comes to mind is batch sorted merge, i.e. you can have efficient point queries on segmentby, and at the same time efficient ordering by orderby. Does it have other uses? For point queries on segmentby, I think the sparse index would be good enough.

Reply:
Yes, if we're not using segmentby, that column should go to orderby. Good point about decompression granularity, but minmax indexes could help there, not sure if we're using them now.